### PR TITLE
Feat/cc 1/empty state

### PR DESCRIPTION
**Pull request by ecni19**
Implemented empty state component.

https://www.figma.com/file/b7geqz0aCeDo4NbzTBWAiw/Empty-states?node-id=198%3A509

Screen Shot 2022-07-11 at 8 44 04 AM

Screen Shot 2022-07-11 at 8 44 50 AM

Screen Shot 2022-07-11 at 8 45 03 AM

### CODE REVIEW

**Comment by owilliams320** [comment]
@ecni19 can you include a gif or some screen shots of what you modified?

File: lib/components/src/empty-state/empty-state.ts [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+        .covalent-icon {
+          font-size: 40px;
+        }
+        .mdc-typography--headline6 {
+          margin: 16px 0 8px 0;
+          font-weight: 600 !important;
+        }
+        .mdc-typography--caption {
+          margin: 0;
+          font-size: 12px;
```
Comment (owilliams320) [inline lib/components/src/empty-state/empty-state.ts:51]: you should be able to pull in css variables here for the typography values. for example this on `var(--mdc-typography-caption-font-size)` you can reference the values here lib/components/styles/theme/_all-theme.scss

**Review by owilliams320 — CHANGES_REQUESTED** [review]
Can you back out these card and data table changes to make this pr a bit more easier to comment on?

**Comment by ecni19** [comment]
@owilliams320 Can you take another time when you have the time. Thanks!

File: lib/components/src/empty-state/empty-state.scss [hunk 2]
```diff
@@ -0,0 +1,48 @@
+@use '@material/typography/mdc-typography';
+
+.content {
+    width: 632px;
```
Comment (owilliams320) [inline lib/components/src/empty-state/empty-state.scss:4]: i dont think we should be setting the width and the height here. can we remove this and if its needed code in story example

**Review by owilliams320 — CHANGES_REQUESTED** [review]
one last comment but overall looks good

**Comment by owilliams320** [comment]
can you rebase with the latest main branch to get the icon fixes?  `git pull --rebase origin main`

**Pull request merged**